### DISCUSSION TITLE
Acurite, RTL_433 and Antennas

### DISCUSSION THREAD

**Discussion by LeeThompson** [merbanan/rtl_433#2731] [Q&A]
# Background
I have 5 acurite sensors in the house: one in the bedroom (second floor), one in the computer room, one in the kitchen (first floor), one in the freezer (garage) and one in the refrigerator (kitchen).  (There's actually a 6th for it's for testing at the moment.)

I have written an application that shows the data pulled from a MariaDB database.   RTL_433 also sends the information to Mosquitto for HomeAssistant.   (For database, I wrote a PHP script that uses the CSV RTL_433 generates to add data).

# Hardware
The hardware is in the computer room which is on the second floor.   

- Production machine is running Windows 10, Development is running Windows 11 (both are Enterprise edition)
- The sensors are all 6002M / 592TX sensors and work fine.
- Receiver is NooElec NESDR Mini 2 SDR & DVB-T USB Stick (I actually have 2, one is on the 'production' machine, and the other is for my 'development' machine)
- The production machine has the antenna that comes with the NooElec NESDR Mini 2, I found attaching it to the wall gave it the best signal.
- The development machine I bought a YOTENKO 10 dBi 433Mhz Antenna Half-Wave Dipole SMA Male Antenna Cable-3M for it's Mini 2.  Ironically the development machine has the most signal issues.

# The problem
My issue is a strange one, over time, a unit will "drop" out of RTL_433's log.  Sometimes if I reset the sensor (disconnect battery and reconnect) it comes back, other times I just need to move the antenna a short distance.     A few times though I end up having to move the antenna a foot or so.

I am pretty much a novice when it comes to radio/electronics so I'm really hoping it's something stupid.

# Signals

These are samples of signal levels, I have a *very basic* understanding of this stuff and could use some help.

When I initially posted this the development machine was not seeing the refrigerator at all, I moved the antenna literally 4 inches and it came back.

| Location | Machine | Frequency | RSSI | SNR | Noise |
| :------ | :------ | :------ | :------  | :------  | :------  |
| Computer Room | Dev | 433.95434 | -2.30163 | 27.28985 | -29.5915 |
| Computer Room | Dev | 433.93322 | -6.46571 | 22.25427 | -28.72 |
| Computer Room | Prod | 433.95802 | -1.84387 | 22.23853 | -24.0824 |
| Computer Room | Prod | 433.95434 | -2.32601 | 20.95328 | -23.2793 |
| Bedroom | Dev | 433.93446 | -6.05466 | 22.86734 | -28.922 |
| Bedroom | Dev | 433.93427 | -6.13012 | 23.00378 | -29.1339 |
| Bedroom | Prod | 433.93341 | -6.38655 | 11.8483 | -18.2348 |
| Bedroom | Prod | 433.93418 | -6.13665 | 8.549 | -14.6856 |
| Kitchen | Dev | 433.98928 | -0.126408 | 28.40051 | -28.5269 |
| Kitchen | Dev | 433.98858 | -0.136242 | 29.22042 | -29.3567 |
| Kitchen | Prod | 433.96986 | -0.104084 | 19.73562 | -19.8397 |
| Kitchen | Prod | 433.97219 | -0.134327 | 23.20173 | -23.3361 |
| Freezer | Dev | 433.98794 | -0.134052 | 27.38617 | -27.5202 |
| Freezer | Dev  | 433.98973 | -0.133781 | 29.00012 | -29.1339 |
| Freezer | Prod | 433.98653 | -0.120678 | 21.85319 | -21.9739 |
| Freezer | Prod | 433.93267 | -6.61995 | 16.71611 | -23.3361 |
| Refrigerator | Dev | 433.92653 | -9.51731 | 10.53173 | -20.049 |
| Refrigerator | Dev | 433.96038 | -1.15638 | 18.29269 | -19.4491 |
| Refrigerator | Prod |  433.93251 | -6.67138 | 16.28204 | -22.9534 |
| Refrigerator | Prod |  433.93341 | -6.36699 | 11.18329 | -17.5503 |

# Environment

This is a two story house with a garage.  The computer room is on the second floor   and is directly over the garage, the kitchen is on the first floor next to the garage.  The bedroom is on the other side of the second floor.

There is a WiFi network on the 2.4, 5 and 6Ghz bands.  The neighbors also have WiFi.   The computer room has... well computers in it as well as 3 NAS (RAID) units.

**Comment by gdt** [comment]
The mini 2 has a plastic case and I think no TCXO.   When I upgraded from that to a SmarTEE or a blog v3, it worked vastly better.   My advice is buy better dongles, and then think about antennas.

**Reply by LeeThompson** [reply]
@gdt Any specific models I should look at?

**Comment by gdt** [comment]
Basically any of the NooElec SmarTee sort of things (metal case, TCXO) and any of the RTL-SDR Blog models (the actual ones, not the counterfeits!) seem good.    There have been recent changes to librtsldr for Blog v4, and v3 should be more broadly compatible with older bits.  But v4 might be better if you want HF -- I'm unclear on the direct sampling operation.

https://www.rtl-sdr.com/buy-rtl-sdr-dvb-t-dongles/

For NooElec, note that many models are in a similar metal case and have a TCXO.   Some have bias T, some have extra range.   You might well not need either.

https://www.nooelec.com/store/sdr/sdr-receivers.html

Also, I use USB extension cables to put the receiver away from the computer (apparently this is particularly important on RPI).

I have tested SmarTEE v2, SmarTEE v5, and Blog v3, and all seemed very good (and similar in performance).   This group is vastly better than the NooElec Mini and NooElec Mini2 (which to be fair are vastly older designs).

I also found that dipoles worked much better, probably because ground plane antennas need a ground plane and don't get them.    So just buying a Blog v3  with the dipole kit, and seeing how that works for you, is what I would recommend.   It's not that I think Blog is better than NooElec, but it's less confusing so it feels easier to give advice.

Of course, actual test data from anyone is really welcome!
